### ISSUE TITLE
[a11y] Screen Reader doesn't announce breadcrumbs properly

### ISSUE THREAD

**Issue by shreyachatterjee00** [civiform/civiform#10122]
The breadcrumbs, on the application form, only visually show as a single link with the Program name.  However, the screen reader announces “Home, link visited, 1 of 3 items” and the focus disappears because there is no “Home” and there is no “Current page, application for…” visible on the page.  There are also not 3 items.

Video:

https://github.com/user-attachments/assets/929eb996-474a-4eea-9b22-fcbb19b7e6b4

**Comment by melanie-exygy** [comment]
Note that this bug also occurs with the USWDS breadcrumb component example.  Whoever picks up this issue should probably file a bug with USWDS.

**Comment by melanie-exygy** [comment]
There are two facets to this issue:
1. Mobile users (without screen reader) have no way to return home by using the breadcrumbs.  They also don't see the current page.  This seems to defeat the purpose of having breadcrumbs.
2. For mobile users with a screen reader, there are links that are announced but not visible on the screen.  This means that focus goes to the edge of the window on those invisible links.  That breaks the Focus Visible WCAG rule:  https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html.

@kpeterson-exygy, would it be acceptable for us to keep the three breadcrumb links visible on mobile, stacking and/or wrapping them so that they fit the screen?  That would solve both of these problems.

**Comment by kpeterson-exygy** [comment]
@melanie-exygy I'm good with wrapping the breadcrumb links. The USWDS guidance does say that might be required for usability. Just noting that we may need to consider doing some text truncation as well depending on how it looks

**Comment by tallulahkay** [comment]
@kpeterson-exygy when this hits staging envs (which I think is tomorrow) can you let us know what you think about if we need to truncate text? @melanie-exygy also mentioned [here](https://github.com/civiform/civiform/pull/10563#discussion_r2089638624) that we could possibly need to ensure a carat never ends up alone on a line. I created [another issue](https://github.com/civiform/civiform/issues/10573) to track any styling follow-ups necessary

**Comment by kpeterson-exygy** [comment]
Thanks @tallulahkay - I'll make sure to take a look
